src/containers/shared/components/Transaction/XChainAccountCreateCommit/Simple.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Amount } from '../../Amount'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+import { XChainBridge } from '../XChainBridge'
+import Account from '../../Account'
+
+export const Simple: TransactionSimpleComponent = (
+  props: TransactionSimpleProps,
+) => {
+  const { t } = useTranslation()
+  const {
+    data: {
+      instructions: {
+        lockingDoor,
+        lockingIssue,
+        issuingDoor,
+        issuingIssue,
+        amount,
+        bridgeOwner,
+        otherChainDestination,
+      },
+    },
+  } = props
+
+  return (
+    <>
+      <SimpleRow label={t('send')} data-test="send">
+        <Amount value={amount} />
+      </SimpleRow>
+      <XChainBridge
```

Comment:
Out of scope but would it be easier to keep these as one value that is passed in.  They seem to always be passed in together.

Reply:
Oh huh I didn't even think about that - yes, I could do that, but I'll save it for another PR.